action.yml

```diff
@@ -75,6 +85,17 @@ runs:
          if ${{ inputs.load-all }} = true then
             LoadAllPackages();
          fi;
+
+         OutputLogTo();
+         CloseStream(output_stream);
+
+         # Treat warnings as errors if specified
+         if ${{ inputs.warnings-as-errors }} = 'true' ; then
```

Comment:
Is that semicolon there legal? Does this all have to be GAP code or is it some weird hybrid?